apps/token-e2e/cypress.config.js

```diff
@@ -6,12 +6,14 @@ module.exports = defineConfig({
     baseUrl: 'http://localhost:4210',
     fileServerFolder: '.',
     fixturesFolder: false,
-    specPattern: [
-      './src/integration/view/**/*.cy.{js,jsx,ts,tsx}',
-      process.env.CYPRESS_INCLUDE_FLOWS
-        ? './src/integration/flow/**/*.cy.{js,jsx,ts,tsx}'
-        : '',
-    ],
+    specPattern:
+      process.env.CYPRESS_INCLUDE_FLOWS === 'true' ||
+      process.env.CYPRESS_INCLUDE_FLOWS === true
```

Comment:
Ignore if this works, I would expect the way to get variable would be `Cypress.env('INCLUDE_FLOWS)`

Reply:
Cypress is not yet defined at this point, so it needs to be handled this way.